src/vcpkg/base/downloads.cpp

```diff
@@ -418,9 +467,11 @@ namespace vcpkg::Downloads
         Checks::check_exit(VCPKG_LINE_INFO, urls.size() > 0);
 
         auto download_path_part_path = download_path;
-        download_path_part_path += fs::u8path(".part");
-        fs.remove(download_path, ignore_errors);
-        fs.remove(download_path_part_path, ignore_errors);
+#if defined(_WIN32)
+        download_path_part_path += fs::u8path(Strings::concat(".", _getpid(), ".part"));
+#else
+        download_path_part_path += fs::u8path(Strings::concat(".", getpid(), ".part"));
+#endif
 
         std::string errors;
         for (const std::string& url : urls)
```

Comment:
Maybe not relevant since this is not part of the PR. If a url is not valid, we may get an unhelpful error message. For example, in line 480 (expand below): `split_uri_view(url)` will print `Value is null` and exit if any url is missing a colon.